factfinder/utils.py

```diff
@@ -50,23 +60,33 @@ def get_z(e, m, p, agg_e, agg_m):
 
 def format_geoid(geoid):
```

Comment:
format geoid and format_geotype should be class methods under the AggregatedGeography class. So that we have room to change geoid and geotype format based on the year of geography